integrationtest/spring/application/src/main/resources/application.yaml

```diff
@@ -0,0 +1,16 @@
+spring.jpa:
+  properties:
+    hibernate:
+      dialect: org.hibernate.dialect.H2Dialect
+    hibernate.search:
+      backend:
+        type: lucene
```

Comment:
This shouldn't be necessary as you only have the Lucene backend in your dependencies: it'll be picked by default.